environment-build.yml

```diff
@@ -0,0 +1,14 @@
+name: cyrsoxs-build
+
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - gcc
```

Comment:
I was getting `gcc: fatal error: cannot execute 'cc1plus': execvp: No such file or directory` following these instructions. It seems to be from a version mismatch between gcc and g++. I replaced `- gcc` with `- cxx-compiler` and was able to build with and without pybind successfully.

Reply:
Done, great catch!